### ISSUE TITLE
AiEnd() and allocated AiArray causes segfault.

### ISSUE THREAD

**Issue by smaragden** [caron/FabricArnold#5]
There seems to be a double deallocation of AiArrays that creates a segfault.

It only happend when arnold AiEnd() is called and an AiArray have been allocated.
Conclusion is that the array is deallocated by the Fabric extension and then arnold is trying to deallocate it again.

Here's a backtrace:
```
*** glibc detected *** python: free(): invalid pointer: 0x00007fffdf1b4218 ***
======= Backtrace: =========
/lib64/libc.so.6[0x3bece76166]
/mcp/usr/local/arnold/4.2.0.4/bin/libai.so(free+0x155)[0x7f3a01d5b295]
/mcp/usr/local/arnold/4.2.0.4/bin/libai.so(AiArrayDestroy+0xbc)[0x7f3a0167623c]
/mcp/usr/local/arnold/4.2.0.4/bin/libai.so(+0x23e1ea)[0x7f3a0181b1ea]
/mcp/usr/local/arnold/4.2.0.4/bin/libai.so(AiEnd+0x76d)[0x7f3a0169f0cd]
[0x7f3a03525952]
```
Deallocation of arrays in extension is here:

https://github.com/caron/FabricArnold/blob/cea1ecd9a50f55929262be6637ad82f7b4e78248/cpp/array.cpp#L35-40

**Comment by caron** [comment]
Hey Fredrik

So Peter Zion has fixed this issue! 3f421abdc669295c20385a03605b2d7071d1a0c9

Closing!

**Comment by smaragden** [comment]
Sweet!

Cheers
/Fredrik
